ci/build-unix.yml

```diff
@@ -36,6 +36,48 @@ steps:
     env:
       DAML_SDK_RELEASE_VERSION: ${{parameters.release_tag}}
 
+  - bash: |
+      set -euo pipefail
+      eval "$(./dev-env/bin/dade-assist)"
+      bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/detect-haskell-snapshot/synopsys-detect) \
+      ci-build digitalasset_daml enable-blackduck \
```

Comment:
@dasormeter change this to `master` or variable that gives current branch before merging. -- hardcoded to enable-blackduck for testing